compiler/rustc_lint_defs/src/lib.rs

```diff
@@ -9,9 +9,10 @@ use rustc_data_structures::stable_hasher::{
     HashStable, StableCompare, StableHasher, ToStableHashKey,
 };
 use rustc_error_messages::{DiagArgValue, DiagMessage, IntoDiagArg, MultiSpan};
-use rustc_hir_id::{HashStableContext, HirId, ItemLocalId};
+use rustc_hir::def_id::DefPathHash;
+use rustc_hir::lints::AttributeLintKind;
+use rustc_hir::{HashStableContext, HirId, ItemLocalId};
 use rustc_macros::{Decodable, Encodable, HashStable_Generic};
-use rustc_span::def_id::DefPathHash;
```

Comment:
Please leave `DefPathHash` imported from `rustc_span`, and `HashStableContext, HirId, ItemLocalId` imported from `rustc_hir_id`, so that only `AttributeLintKind` is imported from `rustc_hir`.